espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/ComposeNodeElement.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.espressoserver.lib.model
+
+import android.graphics.Rect
+import androidx.compose.ui.semantics.*
+
+class ComposeNodeElement(private val node: SemanticsNode) {
+
+    val contentDescription: CharSequence? =
+        node.config.getOrNull(SemanticsProperties.ContentDescription)?.firstOrNull()
```

Comment:
null values should be translated to an empty text for the xml representation

Reply:
same as above

Reply:
yeah, I would say we should reconsider that. At some point of time. Basically my reason is that looking for an empty attribute is simpler and more straighforward in xpath queries in comparison to `no-attribute` queries, e.g.:
`//*[not(@text)]` vs `//*[@text=""]`

Reply:
I was thinking if there is any disadvantage of having lots of empty attributes, can it affect XPath lookup speed while matching specific text values?

Reply:
I assume one won't see the difference in perf unless the page source itself would have hundreds on megabytes in size which will never be the case for what we are doing. Having empty attributes is more preferable from a user perspective, because it is always visible which attribute names could be present there